src/components/organisms/Room/Participant.tsx

```diff
@@ -135,6 +135,7 @@ const Participant: React.FC<React.PropsWithChildren<ParticipantProps>> = ({
           <UserProfilePicture
             user={{ ...profileData, id: participant.identity }}
             setSelectedUserProfile={() => setShowProfile(true)}
+            reactionPosition="right"
```

Comment:
This looked fine when the participant was on both the left of the screen and the right of the screen yeah?

Reply:
For small to medium-sized messages yes

Reply:
Awesome